### PR TITLE
buildah: update to 1.22.3

### PR DESCRIPTION
**Pull request by kwshi**
#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

File: srcpkgs/buildah/template [hunk 1]
```diff
@@ -16,7 +16,7 @@ license="Apache-2.0"
 homepage="https://github.com/containers/buildah"
 changelog="https://github.com/containers/buildah/blob/master/CHANGELOG.md"
 distfiles="${homepage}/archive/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/buildah/template:18]: let's remove the $homepage variable from this $distfiles

**Comment by paper42** [comment]
@CameronNemo ok?

**Comment by CameronNemo** [comment]
LGTM!

**Comment by ericonr** [comment]
1.23 is out, do we want that?

**Comment by CameronNemo** [comment]
LGTM

**Pull request merged**